From f0718d1815ffa81217f687acaffa36947f27adc5 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Sat, 3 Dec 2022 13:03:10 +0100
Subject: [PATCH attr] man: Use apostrophe instead of acute accent
Origin: vendor
Forwarded: yes
Applied-Upstream: 2.5.1+, commit:cbdc7576a832f0d75d100945476cd21f159a2372
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The groff sequence «\'» generates an acute accent, when the intention
here was to use apostrophe.

Warned-by: lintian
Signed-off-by: Guillem Jover <guillem@hadrons.org>
---
 man/man3/attr_list.3   | 8 ++++----
 man/man3/attr_multi.3  | 2 +-
 man/man3/attr_remove.3 | 2 +-
 man/man3/attr_set.3    | 2 +-
 4 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/man/man3/attr_list.3 b/man/man3/attr_list.3
index d7ea3aa..3035b1e 100644
--- a/man/man3/attr_list.3
+++ b/man/man3/attr_list.3
@@ -132,11 +132,11 @@ call should be used to get the value.
 The
 .B attr_list
 interface tells the calling process how large of a buffer
-it must have in order to get the attribute\'s value.
+it must have in order to get the attribute's value.
 .PP
 The
 .I flags
-argument can contain the following symbols bitwise OR\'ed together:
+argument can contain the following symbols bitwise OR'ed together:
 .TP
 .SM
 \%ATTR_ROOT
@@ -159,13 +159,13 @@ The default is to follow symbolic links.
 The
 .I cursor
 argument is a pointer to an opaque data structure that the kernel uses
-to track the calling process\'s position in the attribute list.
+to track the calling process's position in the attribute list.
 The only valid operations on a
 .I cursor
 are to pass it into an
 .B attr_list
 function call or to zero it out.
-It should be zero\'ed out before the first
+It should be zero'ed out before the first
 .B attr_list
 call.
 Note that multi-threaded applications may keep more than one
diff --git a/man/man3/attr_multi.3 b/man/man3/attr_multi.3
index 12aa1d1..f7b5c04 100644
--- a/man/man3/attr_multi.3
+++ b/man/man3/attr_multi.3
@@ -131,7 +131,7 @@ field is initially the total size of the memory buffer that the
 field points to.
 After the operation, the
 .I am_length
-field contains the actual size of the attribute\'s value.
+field contains the actual size of the attribute's value.
 The
 .I am_flags
 field may be set to the \f4ATTR_ROOT\fP flag.
diff --git a/man/man3/attr_remove.3 b/man/man3/attr_remove.3
index 51cc3ce..a3a123b 100644
--- a/man/man3/attr_remove.3
+++ b/man/man3/attr_remove.3
@@ -50,7 +50,7 @@ exists, the attribute name and value will be removed from the
 filesystem object.
 The
 .I flags
-argument can contain the following symbols bitwise OR\'ed together:
+argument can contain the following symbols bitwise OR'ed together:
 .TP
 .SM
 \%ATTR_ROOT
diff --git a/man/man3/attr_set.3 b/man/man3/attr_set.3
index fd72938..6887652 100644
--- a/man/man3/attr_set.3
+++ b/man/man3/attr_set.3
@@ -98,7 +98,7 @@ are set in the same call.
 \%ATTR_REPLACE
 Return an error (ENOATTR) if an attribute of the given name
 does not already exist on the indicated filesystem object,
-otherwise replace the existing attribute\'s value with the given value.
+otherwise replace the existing attribute's value with the given value.
 This flag is used to implement a pure replacement operation,
 without this flag
 .B attr_set
-- 
2.38.1

